### ISSUE TITLE
[Accessibility] Keyboard user is unable to collapse the Country flag Combo box  by using "Esc" key

### ISSUE THREAD

**Issue by mayukh-ms** [catamphetamine/react-phone-number-input#320]
Keyboard user is unable to collapse the Country flag Combo box  by using "Esc" key.

**Package version:** 2.3.5
**OS version:** Window 10
**Browser:** Microsoft Edge 44.18362.1.0

**Comment by catamphetamine** [comment]
Hi.
I've released v3 of the library.
See if the demo works, and if not — specify a more detailed description of the issue you're having.

I'm able to close the country select on the demo page on Esc key.

**Comment by catamphetamine** [comment]
Actually, I re-checked it in Edge and I also observe that it doesn't close the country select on Esc key.
This seems to be the behavior of any `<select/>` in Edge.
For example, on this page the `<select/>` also doesn't collapse on Esc key:
https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_select_create
So this seems to be an MS Edge bug.
